lib/omnibus/packagers/base.rb

```diff
@@ -177,11 +177,11 @@ def skip_packager(val = false)
     #   - setup
     #   - build
     #
-    def run!
+    def run!(duration_cb = nil)
       # Ensure the package directory exists
       create_directory(Config.package_dir)
 
-      measure("Packaging time") do
+      measure("Packaging time", duration_cb) do
```

Comment:
```suggestion
      measure("Packaging time", ->(duration) { project.package_summary[id] = duration }) do
```

That would require making `project.package_summary` public or adding some sort of explicit setter method such as `project.record_duration(:packaging, id, duration)`.

Similarly for the `stripper`.

Reply:
I think I understood your proposal and I like it better that the current one. I'll attempt to implement it asap
Thanks for the idea!